config/locales/sv.yml

```diff
@@ -79,6 +82,14 @@ sv:
         email: &email E-post
         password: *password
         created: &created Skapad
+        current_sign_in_at: &user_current_signed_in_time Inloggad pä
```

Comment:
I did find some issues :) 
I think pä is supposed to be på. But ut can be omitted. 

So, line 85: "Inloggad"